### ISSUE TITLE
Push Compensation option

### ISSUE THREAD

**Issue by succeed909** [inkstitch/inkstitch#3322]
Push/Pull Compensation 这个功能有吗？一直找不到！如果有可以调数值吗？
![p](https://github.com/user-attachments/assets/ed744be2-051a-464d-a944-3d656ba09687)

**Comment by kaalleen** [comment]
You either digitize your shape accordingly or for fills you can add pull compensation through the params dialog.
https://inkstitch.org/docs/stitches/fill-stitch/#params

**Comment by succeed909** [comment]
> 您可以相应地数字化您的形状，或者对于填充，您可以通过 params 对话框添加拉动补偿。 https://inkstitch.org/docs/stitches/fill-stitch/#params
> 
> 当然，对于缎子来说，这也是可能的：https://inkstitch.org/docs/stitches/satin-column/#params

Yes. But they only have a pull compensation function, not a push compensation function.

> You either digitize your shape accordingly or for fills you can add pull compensation through the params dialog. https://inkstitch.org/docs/stitches/fill-stitch/#params
> 
> For satins of course that is possible too: https://inkstitch.org/docs/stitches/satin-column/#params

yes. But they only have a pull compensation function, not a push compensation function
